frontend/src/components/VTag/VTag.vue

```diff
@@ -4,8 +4,10 @@
     size="small"
     variant="filled-gray"
     class="label-bold"
-    :href="href"
-    >{{ title }}</VButton
+    v-bind ="$props"
+    v-on="$listeners"
+    aria-label="Tag"
+    ><slot></slot></VButton
```

Comment:
```suggestion
    ><slot /></VButton
```

We usually use the self-closing tag if there's no content.